nemo/collections/tts/torch/tts_tokenizers.py

```diff
@@ -123,16 +84,18 @@ class EnglishCharsTokenizer(BaseTokenizer):
 
     def __init__(
         self,
+        chars,
         punct=True,
         apostrophe=True,
         add_blank_at=None,
         pad_with_space=False,
         non_default_punct_list=None,
-        text_preprocessing_func=english_text_preprocessing,
-        word_tokenize_func=english_word_tokenize,
+        text_preprocessing_func=lambda x: x,
```

Comment:
Does it still make sense to have this function if we split normalization and tokenization?